src/core/tests/frontend/onnx/op_extension.cpp

```diff
@@ -0,0 +1,144 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "op_extension.hpp"
+
+#include "onnx_utils.hpp"
+#include "openvino/frontend/extension/op.hpp"
+#include "openvino/frontend/onnx/extension/op.hpp"
+#include "openvino/frontend/onnx/frontend.hpp"
+#include "so_extension.hpp"
+
+using namespace ov::frontend;
+
+using ONNXOpExtensionTest = FrontEndOpExtensionTest;
+
+static const std::string translator_name = "Relu";
+
+class Relu1 : public Relu {
+public:
+    OPENVINO_OP("CustomRelu_1");
+    OPENVINO_FRAMEWORK_MAP(onnx)
+};
+
+class Relu2 : public Relu {
+public:
+    OPENVINO_FRAMEWORK_MAP(onnx, "CustomRelu_2")
```

Comment:
Should you add `OPENVINO_OP` also?

Reply:
not necessary, in this case we will use the base class name for OpenVINO op and "CustomRelu_2" for framework op to handle it in FrontEnd